metadata/src/main/java/org/apache/kafka/controller/ConfigurationControlManager.java

```diff
@@ -260,6 +279,7 @@ private ApiError incrementalAlterConfigResource(
         if (error.isFailure()) {
             return error;
         }
+        maybeTriggerPartitionUpdateOnMinIsrChange(newRecords);
```

Comment:
we need to support `legacyAlterConfigResource` also.

Reply:
`legacyAlterConfigResource` has supported. Adding UT to cover it.